### ISSUE TITLE
Error after turning off L2TP VPN connection - Ubuntu 16.10

### ISSUE THREAD

**Issue by RafLeszczynski** [xelerance/xl2tpd#128]
After turning off L2TP VPN connection, I'm having System reported and Error dialog. Please check logs form `var/log/syslog`. Also I'm not sure if this is related but Google Chrome remembers VPN location and keeps default search page related do this location (changing default search engine is Chrome settings fix this issue). Running Ubuntu 16.10.

I'm pretty new to linux. Any idea what's the root cause of the problem?

```
Mar 27 13:25:08 ubuntu-desktop NetworkManager[1174]: <info>  [1490613908.2511] manager: NetworkManager state is now CONNECTED_LOCAL
Mar 27 13:25:08 ubuntu-desktop whoopsie[1141]: [13:25:08] offline
Mar 27 13:25:08 ubuntu-desktop NetworkManager[1174]: <info>  [1490613908.2556] audit: op="connection-deactivate" uuid="19df092e-18ec-4ff6-9971-2b6ab8858e25" name="kodify.io" pid=4486 uid=1000 result="success"
Mar 27 13:25:08 ubuntu-desktop NetworkManager[1174]: <info>  [1490613908.2576] manager: NetworkManager state is now CONNECTED_GLOBAL
Mar 27 13:25:08 ubuntu-desktop NetworkManager[1174]: <info>  [1490613908.2581] dns-mgr: Writing DNS information to /sbin/resolvconf
Mar 27 13:25:08 ubuntu-desktop dnsmasq[2229]: setting upstream servers from DBus
Mar 27 13:25:08 ubuntu-desktop dnsmasq[2229]: using nameserver 192.168.2.2#53(via wlp4s0)
Mar 27 13:25:08 ubuntu-desktop charon: 14[KNL] interface ppp1 deactivated
Mar 27 13:25:08 ubuntu-desktop dbus[1153]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service'
Mar 27 13:25:08 ubuntu-desktop NetworkManager[1174]: nm-l2tp[22353] <info>  Terminated xl2tpd daemon with PID 22427.
Mar 27 13:25:08 ubuntu-desktop NetworkManager[1174]: xl2tpd[22427]: death_handler: Fatal signal 15 received
Mar 27 13:25:08 ubuntu-desktop charon: 15[KNL] 10.0.0.250 disappeared from ppp1
Mar 27 13:25:08 ubuntu-desktop kernel: [17163.713369] xl2tpd[22427]: segfault at 188 ip 000000000040bd08 sp 00007ffc058fdc30 error 4 in xl2tpd[400000+1b000]
Mar 27 13:25:08 ubuntu-desktop whoopsie[1141]: [13:25:08] Could not determine if /org/freedesktop/NetworkManager/ActiveConnection/3 is a default route:
Mar 27 13:25:08 ubuntu-desktop whoopsie[1141]: [13:25:08] GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such interface 'org.freedesktop.DBus.Properties' on object at path /org/freedesktop/NetworkManager/ActiveConnection/3
Mar 27 13:25:08 ubuntu-desktop whoopsie[1141]: [13:25:08] Could not determine if /org/freedesktop/NetworkManager/ActiveConnection/3 is a default route:
Mar 27 13:25:08 ubuntu-desktop whoopsie[1141]: [13:25:08] GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such interface 'org.freedesktop.DBus.Properties' on object at path /org/freedesktop/NetworkManager/ActiveConnection/3
Mar 27 13:25:08 ubuntu-desktop whoopsie[1141]: [13:25:08] The default IPv4 route is: /org/freedesktop/NetworkManager/ActiveConnection/1
Mar 27 13:25:08 ubuntu-desktop whoopsie[1141]: [13:25:08] Not a paid data plan: /org/freedesktop/NetworkManager/ActiveConnection/1
Mar 27 13:25:08 ubuntu-desktop whoopsie[1141]: [13:25:08] Found usable connection: /org/freedesktop/NetworkManager/ActiveConnection/1
Mar 27 13:25:08 ubuntu-desktop NetworkManager[1174]: <info>  [1490613908.2813] vpn-connection[0x5629773fb410,19df092e-18ec-4ff6-9971-2b6ab8858e25,"kodify.io",0]: VPN plugin: state changed: stopping (5)
Mar 27 13:25:08 ubuntu-desktop NetworkManager[1174]: Stopping strongSwan IPsec...
Mar 27 13:25:08 ubuntu-desktop charon: 00[DMN] signal of type SIGINT received. Shutting down
Mar 27 13:25:08 ubuntu-desktop systemd[1]: Starting Network Manager Script Dispatcher Service...
Mar 27 13:25:08 ubuntu-desktop charon: 00[IKE] closing CHILD_SA nm-ipsec-l2tp-22353{1} with SPIs cad0eace_i (2261568 bytes) 04c825b2_o (298910 bytes) and TS 192.168.2.78/32[udp/l2f] === 81.47.131.124/32[udp/l2f]
Mar 27 13:25:08 ubuntu-desktop charon: 00[IKE] sending DELETE for ESP CHILD_SA with SPI cad0eace
Mar 27 13:25:08 ubuntu-desktop charon: 00[ENC] generating INFORMATIONAL_V1 request 1226276165 [ HASH D ]
Mar 27 13:25:08 ubuntu-desktop charon: 00[NET] sending packet: from 192.168.2.78[4500] to 81.47.131.124[4500] (76 bytes)
Mar 27 13:25:08 ubuntu-desktop charon: 00[IKE] deleting IKE_SA nm-ipsec-l2tp-22353[1] between 192.168.2.78[192.168.2.78]...81.47.131.124[81.47.131.124]
Mar 27 13:25:08 ubuntu-desktop charon: 00[IKE] sending DELETE for IKE_SA nm-ipsec-l2tp-22353[1]
Mar 27 13:25:08 ubuntu-desktop charon: 00[ENC] generating INFORMATIONAL_V1 request 804183604 [ HASH D ]
Mar 27 13:25:08 ubuntu-desktop charon: 00[NET] sending packet: from 192.168.2.78[4500] to 81.47.131.124[4500] (84 bytes)
Mar 27 13:25:08 ubuntu-desktop dbus[1153]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Mar 27 13:25:08 ubuntu-desktop systemd[1]: Started Network Manager Script Dispatcher Service.
Mar 27 13:25:08 ubuntu-desktop nm-dispatcher: req:1 'vpn-down' [ppp1]: new request (1 scripts)
Mar 27 13:25:08 ubuntu-desktop nm-dispatcher: req:1 'vpn-down' [ppp1]: start running ordered scripts...
Mar 27 13:25:08 ubuntu-desktop systemd[3603]: Starting Notification regarding a crash report...
Mar 27 13:25:08 ubuntu-desktop nm-l2tp-service[22353]: ipsec shut down
Mar 27 13:25:08 ubuntu-desktop NetworkManager[1174]: <info>  [1490613908.3803] vpn-connection[0x5629773fb410,19df092e-18ec-4ff6-9971-2b6ab8858e25,"kodify.io",0]: VPN plugin: state changed: stopped (6)
Mar 27 13:25:08 ubuntu-desktop update-notifier-crash[23777]: xl2tpd
Mar 27 13:25:08 ubuntu-desktop system-crash-no[23786]: GtkDialog mapped without a transient parent. This is discouraged.
Mar 27 13:25:09 ubuntu-desktop whoopsie[1141]: [13:25:09] online
Mar 27 13:25:26 ubuntu-desktop systemd[3603]: Started Notification regarding a crash report.
```

**Comment by dkosovic** [comment]
Pretty sure this is related to the following :
* https://bugs.launchpad.net/ubuntu/+source/xl2tpd/+bug/1509183
* https://bugs.launchpad.net/ubuntu/+source/xl2tpd/+bug/1677990

Updated xl2tpd packages with the patch have been released for Ubuntu 16.04 and 16.10 were released in May

**Comment by shussain** [comment]
@RafLeszczynski  If the issue still occurs, please reopen this issue
